lib/models/room.js

```diff
@@ -185,6 +191,18 @@ Room.prototype.getAvatarUrl = function(baseUrl, width, height, resizeMethod,
  };
 
  /**
+  * Get the implicit room name (i.e. what a given user would see if the
+  * room had no m.room.name)
+  * @param {string} userId The userId from whose perspective we want
+  * to calculate the implicit name
+  * @return {string} The implicit room name
+  */
+ Room.prototype.getImplicitRoomName = function(userId) {
```

Comment:
Okay, I understand now.

> I thought 'implicitRoomName' - i.e. the implicit name a room would have if a user hasn't specified an explicit one was fine terminology personally. Do you have a better suggestion?

Looking from your explanation of what you're using this for:

> to show what a room would be called if you try to delete its name

I'm tempted to just call this `Room.getDefaultName()`. People are happy with the idea of "default" values (without something explicitly set) so I'd prefer to piggyback off that rather than "implicit" which isn't used anywhere near as much and is more subjective IMO.